engine/src/test/java/org/camunda/bpm/engine/test/api/history/removaltime/HistoricRootProcessInstanceTest.java

```diff
@@ -588,9 +592,11 @@ public void shouldResolveUserOperationLog_ClaimTask() {
 
     ProcessInstance processInstance = runtimeService.startProcessInstanceByKey(CALLING_PROCESS_KEY);
 
+    String taskId = taskService.createTaskQuery().singleResult().getId();
+
     // when
     identityService.setAuthenticatedUserId("aUserId");
-    taskService.claim(taskService.createTaskQuery().singleResult().getId(), "aUserId");
+    taskService.setAssignee(taskId, "aUserId");
```

Comment:
Why was this test changed?

Reply:
* An already assigned task cannot be claimed. Consider these changes [1]
* The purpose of the test is covering the identity link behavior in general; It does not really matter which API method is called to create the identity link
* My proposal would be to rename the test method

[1] https://github.com/camunda/camunda-bpm-platform/pull/740/files/3db981226a34ce7701440a08a49e5e01c27339ef..5b3ceaf2631992fa79ee0a57562ffe6f5f024de2#diff-c557285d6812e516c93a0a07f4b309c3R75-R77

Reply:
I see, thanks for clarifying. Renaming would be good then, I agree.